cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala

```diff
@@ -25,7 +25,14 @@ import org.clapper.argot.ArgotConverters._
 import org.clapper.argot.{ArgotParser, ArgotUsageException}
 
 object Wadl2Checker {
-  val parser = new ArgotParser("wadl2checker", preUsage=Some("wadl2checker: Version 1.0.0-SNAPSHOT"))
+
+  val creatorString = {
+    val title = getClass.getPackage.getImplementationTitle
+    val version = getClass.getPackage.getImplementationVersion
+    s"$title ($version)"
+  }
+
+  val parser = new ArgotParser("wadl2checker", preUsage=Some(creatorString))
```

Comment:
Is this right or does this `creatorString` replace the `ArgotParser` completely?